lib/task-data/tasks/install-esx.js

```diff
@@ -4,7 +4,7 @@
 
 module.exports = {
     friendlyName: 'Install Esx',
-    injectableName: 'Task.Os.Install.Esx',
+    injectableName: 'Task.Os.Install.ESXi',
```

Comment:
I think other places need also be changed:
-  The filename.
- the kickstart/boot-config file name.
- the osType.
- the properties.os.hypervisor.